src/register/Registration.tsx

```diff
@@ -28,7 +28,7 @@ const Registration = () => {
                 <CustomInput label='Подтвердите пароль' onChange={setAcceptedPassword} />
                 <CustomInput label='Введите номер телефона' onChange={setPhoneNumber} />
                 <CustomInput label='Укажите  имя пользователя' onChange={setUserName} />
```

Comment:
Может, пробел лишний?
```suggestion
                <CustomInput label='Укажите имя пользователя' onChange={setUserName} />
```

Reply:
Тут мне похуй, не мое

Reply:
Ну так передай тому, чье это

Reply:
Это Мишино, пиздежь

Reply:
А реально, паста сломалась, пофиксим

Reply:
Продукт общий - ответственность общая